docker-compose.yml

```diff
@@ -1,15 +1,8 @@
-version: '3.7'
+version: "3.7"
 services:
-  node14:
+  node:
     build:
       context: .
       args:
-        NODE_VERSION: 14
-    image: nodejs:14
-
-  node16:
-    build:
-      context: .
-      args:
-        NODE_VERSION: 16
-    image: nodejs:16
+        NODE_VERSION: ${NODE_VERSION}
```

Comment:
Perhaps set a default value

Reply:
But you can always run `NODE_VERSION=20` docker-compose build or something.

Reply:
Yeah, doesn't matter, just nice to have